src/main/java/Application.java

```diff
@@ -0,0 +1,62 @@
+package main.java;
+
+import java.util.Scanner;
+
+class Operations {
+
+    static void addTwoNumbers(double value1, double value2) {
```

Comment:
it's not necessary to fix by it may look a bit cluttered. It obvious the method which takes two values (as a params) will add them. so it's not necessary to add 'two numbers' to the method's name. Same for other